### ISSUE TITLE
hey facing isssue in installation please make a video

### ISSUE THREAD

**Issue by ayushkkkkk** [developerrahulofficial/jennie-anime-player#2]
just tell me how to install and don't ignore

**Comment by rudransh0999** [comment]
ya me too having the problem.      please make a video on this

**Comment by sushantpatil03** [comment]
same here

**Comment by sushantpatil03** [comment]
TS1261: Already included file name 'D:/Jennie Anime Player/LiSA/src/screens/DownloadsScreen.tsx' differs from file name 'D:/Jennie Anime Player/LiSA/src/screens/downloadsScreen.tsx' only in casing.
  The file is in the program because:
    Imported via "./screens/DownloadsScreen" from file 'D:/Jennie Anime Player/LiSA/src/App.tsx'
    Root file specified for compilation
    13 | import { HomeScreen } from "./screens/homeScreen";
    14 | import { ExploreScreen } from "./screens/exploreScreen";
  > 15 | import { DownloadScreen } from "./screens/DownloadsScreen";
       |                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    16 | import { MyListScreen } from "./screens/MyListScreen";
    17 | import SettingScreen from "./screens/settingScreen";
    18 |


Packaging app for platform win32 x64 using electron v26.2.2
ENOENT: no such file or directory, lstat 'D:\Jennie Anime Player\LiSA\resources\LiSA'

**Comment by domabyte** [comment]
Yes, you guys need to make some changes in order to run the application. 
Replace this line `import { DownloadScreen } from "./screens/DownloadsScreen"` with `import { DownloadScreen } from "./screens/downloadsScreen";` in app.tsx and avoid to run the application in linux for now

**Comment by vtvishwanathmsft** [comment]
I get 503 Service not available error in the powershell

**Comment by Abhinav-coder02** [comment]
Give us direct link to download the application

**Comment by Cosmicoppai** [comment]
@ayushkkkkk @Abhinav-coder02 I doubt he'll be able to resolve any issue, cuz he's just a full time theif, claiming someone else's work.
Look at the original repo:
https://github.com/CosmicOppai/LiSA
